### ISSUE TITLE
VoiceOver does not announce document statistics

### ISSUE THREAD

**Issue by dusek** [textmate/bugs#1]
When user presses ⌃⇧N to read document statistics (Bundles > Text > Statistics for Document (Word Count)), Voiceover should announce the statistics (i.e. read "21 lines, 66 words, 959 characters"). Instead, VoiceOver, after a while announces that "TextMate has new window". Then the user has to do VO-F2-F2 and arrow down to read the window's contents.

This could be improved using the `NSAccessibilityAnnouncementRequestedNotification`.
